docs/whichkey/extra.md

```diff
@@ -44,6 +44,36 @@ You can selectively override that color in your `settings.json` like the followi
 },
 ```
 
+## Styling the menu with Customize UI
+
+VS Code unfortunately only allows for very limited styling of the QuickPick used by `which-key` to display keys.
+However, using the [Customize UI](https://marketplace.visualstudio.com/items?itemName=iocave.customize-ui) extension,
+the style sheet of the VS Code installation can be modified to result in a nicer appearance and better usability.
+
+Suggested settings:
+
+```jsonc
+    "customizeUI.stylesheet": {
+        // Let the quick pick take the full window height, so that more bindings
+        // are visible.
+        ".quick-input-widget > .quick-input-list > .monaco-list":
+            "max-height: none !important;",
+        // Colorize icons uniformly and add some extra padding
+        ".quick-input-widget > .quick-input-list .codicon":
+            "color: #83a598 !important; padding-right: 5px;",
+        // Colorize entries starting with a `+` differently
+        ".quick-input-widget > .quick-input-list .monaco-list-row[aria-label*=\" +\"] .label-description":
+            "color: #8ec07c;",
+    }
```

Comment:
```suggestion
"customizeUI.stylesheet": {
    // Let the quick pick take the full window height, so that more bindings
    // are visible.
    ".quick-input-widget > .quick-input-list > .monaco-list":
        "max-height: none !important;",
    // Colorize icons uniformly and add some extra padding
    ".quick-input-widget > .quick-input-list .codicon":
        "color: #83a598 !important; padding-right: 5px;",
    // Colorize entries starting with a `+` differently
    ".quick-input-widget > .quick-input-list .monaco-list-row[aria-label*=\" +\"] .label-description":
        "color: #8ec07c;",
}
```

Remove the extra indentation?

Reply:
The indentation is deliberate, so that this can be copied 1:1 into the config file - would you still prefer to remove it?

Reply:
Ah. Got you. I don't think we have extra indentation for code block elsewhere. Why don't we try to stay consistent as long as we mentioned this is `settings.json`